ui/components/configuratorComponents/patternConfigurator.js

```diff
@@ -199,14 +199,16 @@ function PatternConfiguratorComponent({ pattern, onSubmit, show : setSelectedPat
   };
 
   const handleSettingsChange = (schemaSet) => () => {
-    const config = createPatternFromConfig({
-      [getPatternServiceName(schemaSet)] : {
-        // @ts-ignore
+    const cfg = {
+      [(Math.random() + 1).toString(36).substring(2)] : {
```

Comment:
https://user-images.githubusercontent.com/48255244/146363435-72891987-15f9-4a99-97d4-32f5bd6bb919.mp4

Reply:
@Abhishek-kumar09 this was intentional because patterns usually do require services of same type. Refer: https://github.com/service-mesh-patterns/service-mesh-patterns/pull/20

Reply:
Yeah, gotcha. Thanks @utkarsh-pro